static/js/place/dev_place_main.tsx

```diff
@@ -413,6 +423,16 @@ const RelatedPlaces = (props: {
           ))}
         </div>
       </div>
+      {showToggle && (
+        <div className="show-more-toggle" onClick={toggleShowMore}>
+          <span className="show-more-toggle-text">
+            {isCollapsed ? "Show more" : "Show less"}
```

Comment:
Can you add the count for show more? e.g., "Show 25 more"

Reply:
Great idea, done! Also updated screenshots in the PR description.